service-api/src/main/java/greencity/dto/employee/UpdateEmployeeAuthoritiesDto.java

```diff
@@ -0,0 +1,18 @@
+package greencity.dto.employee;
+
+import greencity.dto.position.PositionDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
```

Comment:
Is it required to have setters and builders?

Reply:
done